### ISSUE TITLE
Reusing pretrained model in my own model

### ISSUE THREAD

**Issue by yadyad91** [tensorflow/tensorflow#21661]
Hello,
I am creating a tensorflow model with a new session and you will find below the pseudocode for it

```
inp = tf.placeholder()   #----- placeholder for model1
# conv and relu layers
intermediate = conv2d()  # ----- an intermediate output that I want to use in model2
# conv and relu layers
loss = tf.reduce_mean( sigmoid_cross_entropy_with_logits( predicted , real ) )

sess1.run( loss , feed_dict = { inp : images } )
```
Now I want to use a pretrained model and add the output of the final layer in the loss function. So for example,
```

sess2 = tf.Session()
load_model( sess2 )
```
The pretrained model has its own placeholders and I would like to send the value in my intermediate tensor to the placeholder of the pretrained model. Thus, I want to run the pretrained model in every batch and evaluate its prediction.

The problem is that on searching I found out that we cannot feed_dict tensors to placeholders. Is there any other way I can go about and evaluate the second model everytime the first model learns? I asked this question on Stackoverflow and haven't received any responses.

Thanks,

**Comment by yadyad91** [comment]
Here is some additional info:
Have I written custom code: Yes
OS Platform and Distribution: Ubuntu
TensorFlow installed from: pip
TensorFlow version: 1.9
Bazel version: 1.9
CUDA/cuDNN version: CUDA 9, cuDNN 7
GPU model and memory: Quadro p5000
Exact command to reproduce: N/A
Mobile device: N/A

**Comment by bignamehyp** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
